dashboard/test/ui/features/contractEditorSections.feature

```diff
@@ -6,7 +6,6 @@ Background:
   Given I am on "http://studio.code.org/s/algebra/stage/7/puzzle/4?noautoplay=true"
   And I rotate to landscape
   And I wait for the page to fully load
-  And I close the React alert
```

Comment:
Are the uses of this step in contractEdtior.feature referring to a different alert?

If not, can we remove those and the [related step](https://github.com/code-dot-org/code-dot-org/blob/8e544d64b71356d3bef28f6810f38276b75ed568/dashboard/test/ui/features/step_definitions/steps.rb#L118) as well?

https://github.com/code-dot-org/code-dot-org/blob/8e544d64b71356d3bef28f6810f38276b75ed568/dashboard/test/ui/features/step_definitions/steps.rb#L118

Reply:
👍